### PR TITLE
Flatten gateway spec and reword abstract

### PR DESCRIPTION
**Pull request by no-reply**
Related to #14

### CODE REVIEW

File: otm-gateway.html [hunk 1]
```diff
@@ -38,8 +38,9 @@
       </p>
       <p>
         The preservation Gateway functions as an aggregating cache for preservation requests originating with a repository
-        and destined for a DDP via the  Bridge. This allows repositories to synchronously send content preservation in an
-        asynchronous deep archive.
+        and destined for a DDP via the Bridge. The API provides a synchronous interface for requests related to preservation
+        of content, even when the preservation system's interactions are fundamentally asynchronous.
+        in
```
Comment (bbranan) [inline otm-gateway.html]: I expect including this line was not intentional

Comment (bbranan) [reply otm-gateway.html]: The "in" on line 43, that is.

Comment (no-reply) [inline otm-gateway.html]:
```suggestion
```

**Comment by no-reply** [comment]
i have some further work to push through that will necessarily conflict with this (since this changes more or less the entire file). if there aren't major concerns, it would help things move forward to get this one through.

**Review by bbranan — APPROVED** [review]

**Pull request merged**